packages/files-ui/cypress/tests/file-preview-spec.ts

```diff
@@ -0,0 +1,76 @@
+import { homePage } from "../support/page-objects/homePage"
+import { previewModal } from "../support/page-objects/modals/previewModal"
+
+describe("File Preview", () => {
+
+  context("desktop", () => {
+
+    it("can preview and browse compatible file types", () => {
+      cy.web3Login({ clearCSFBucket: true, clearTrashBucket: true })
```

Comment:
We don't need to clear the trash here I guess since it's not used.
```suggestion
      cy.web3Login({ clearCSFBucket: true })
```

Reply:
@Tbaut Ah yes, I added this before I'd clarified bin files were not supposed to show in search and then forgot to remove it. Thanks